src/constants.h

```diff
@@ -1284,6 +1295,16 @@ const std::initializer_list<STELLAR_TYPE> ALL_HERTZSPRUNG_GAP = {
 };
 
 
+// (convenience) initializer list for GIANTS (excluding HG)
+const std::initializer_list<STELLAR_TYPE> GIANTS = {
+    STELLAR_TYPE::FIRST_GIANT_BRANCH,
+    STELLAR_TYPE::CORE_HELIUM_BURNING,
```

Comment:
That's true, I was maybe a bit sloppy with this one. This is a relic from an earlier attempt to setup the critical mass ratios. I didn't end up using this, but figured it might be useful to others in the future. If you think it is more misleading than helpful, I can remove it.

Reply:
Yes, I think it is more misleading than helpful, particularly if you don't use it. Fully remove to avoid any potential problems.